packages/exampleReactNative/App.tsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * Generated with the TypeScript template
+ * https://github.com/react-native-community/react-native-template-typescript
+ *
+ * @format
+ */
+
+import React, { useEffect, useState } from 'react';
+import {
+  SafeAreaView,
+  StyleSheet,
+  ScrollView,
+  View,
+  Text,
+  StatusBar,
+  Button,
+  Picker,
+} from 'react-native';
+
+import {
+  Colors,
+} from 'react-native/Libraries/NewAppScreen';
+
+import Identicon from "@polkadot/reactnative-identicon";
+import settings from '@polkadot/ui-settings';
+import { mnemonicGenerate, cryptoWaitReady } from '@polkadot/util-crypto';
+import keyring from '@polkadot/ui-keyring';
+
+const App = () => {
+  const [ready, setReady] = useState(false)
```

Comment:
Please use semi-standard. (I know it is not needed, but it just aligns with the code conventions)